src/proteus/schemas/dswx_hls.yaml

```diff
@@ -79,8 +79,10 @@ runconfig:
             # Define how areas adjacent to cloud/cloud-shadow should be handled
             mask_adjacent_to_cloud_mode: enum('mask', 'ignore', 'cover', required=False)
 
-            # Copernicus CGLS Land Cover 100m forest classes
-            copernicus_forest_classes: list(int(), min=0, required=False)
+            # Copernicus CGLS Land Cover 100m forest classes to mask out from
```

Comment:
If this is a correct statements, please add a note here (and everywhere else that it is relevant) that class 201 (evergreen) will always be masked out. Thanks!